seatunnel-connectors-v2/connector-clickhouse/src/main/java/org/apache/seatunnel/connectors/seatunnel/clickhouse/source/ClickhouseSourceFactory.java

```diff
@@ -62,65 +39,11 @@ public String factoryIdentifier() {
         return "Clickhouse";
     }
 
-    @Override
-    public <T, SplitT extends SourceSplit, StateT extends Serializable>
-            TableSource<T, SplitT, StateT> createSource(TableSourceFactoryContext context) {
```

Comment:
why?

Reply:
Do the conf parsing directly in ClickhouseSource.prepare to avoid unnecessary parameter passing

Reply:
prepare already marked as deprecated.

https://github.com/apache/seatunnel/blob/dev/seatunnel-api/src/main/java/org/apache/seatunnel/api/common/SeaTunnelPluginLifeCycle.java#L42